src/app/master/master.component.ts

```diff
@@ -27,7 +27,8 @@ export class MasterComponent implements OnInit, OnDestroy {
         this.route.params.subscribe(params => {
             this.themeName = params['theme'] || '';
             this.colorScheme = params['color-scheme'] || '';
-            this.themeSize = params['color-scheme'].split('-')[1];
+            this.isThemeCompact = this.colorScheme.includes('compact');
```

Comment:
IE has no support for `includes`

Reply:
includes method there is not only here, fix this in another PR